src/ssh/test-sshbridge.c

```diff
@@ -289,26 +290,31 @@ setup (TestCase *tc,
 #if LIBSSH_085
   if (fixture->test_home_ssh_config)
     {
-      gchar *content;
-      gchar **host_port;
+      g_autoptr(GString) content = g_string_new (NULL);
+      g_auto(GStrv) host_port;
 
       tc->home_ssh_config_file = g_build_filename (tc->home_ssh_dir, "config", NULL);
       if (!fixture->knownhosts_home)
           g_assert_cmpint (mkdir (tc->home_ssh_dir, 0700), ==, 0);
       host_port = g_strsplit (host, ":", 0);
```

Comment:
This splitting is pretty weird when we have `tc->ssh_port` already split out and we did the pasting together ourselves just above.

Maybe split out the handling of localhost/127.0.0.1 above into a separate variable and use it from here as well.  We already duplicate that logic twice, so let's make it 1 rather than 3.